library/src/doo/shell.clj

```diff
@@ -68,7 +74,7 @@
    as it happens by default."
   ([cmd] (sh cmd {:verbose true}))
   ([cmd opts]
-   (let [proc (exec! cmd)
+   (let [proc (exec! cmd (:base-dir opts))
```

Comment:
You could switch on whether `:base-dir` is present or not to make what's happening more explicit here.